ext/pcntl/tests/pcntl_unshare_04.phpt

```diff
@@ -1,21 +1,38 @@
 --TEST--
-pcntl_unshare() with wrong flag
+pcntl_unshare() - Error with wrong flag. Test return value type for each valid flag.
 --EXTENSIONS--
 pcntl
-posix
 --SKIPIF--
 <?php
 if (!function_exists("pcntl_unshare")) die("skip pcntl_unshare is not available");
 ?>
 --FILE--
 <?php
 
+$result = null;
+
 try {
-    pcntl_unshare(42);
+    $result = pcntl_unshare(-1);
 } catch (ValueError $exception) {
     echo $exception->getMessage() . "\n";
 }
 
+var_dump($result);
+
+foreach ([CLONE_NEWNS, CLONE_NEWIPC , CLONE_NEWUTS , CLONE_NEWNET, CLONE_NEWPID ,CLONE_NEWUSER, CLONE_NEWCGROUP] as $flag) {
+    $result = @pcntl_unshare($flag);
```

Comment:
I understand why you've silenced the call, but I don't think it makes sense to execute the call in the first place. We actually don't test anything here, and the possibly increased code coverage is deceptive.